pkgs/development/libraries/libcs50/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libcs50";
+  version = "11.0.2";
+
+  src = fetchFromGitHub {
+    owner = "cs50";
+    repo = "libcs50";
+    rev = finalAttrs.version;
+    hash = "sha256-A4CEU5wfwykVTDIsKZnQ8co+6RwBGYGZEZxRFzQTKBI=";
+  };
+
+  installPhase = ''
+    mkdir $out
```

Comment:
Missing hooks.

Check this derivation as example on how to add them: https://github.com/NixOS/nixpkgs/pull/234428